nrf_security/Kconfig

```
@@ -838,6 +838,14 @@ if NRF_SECURITY_ADVANCED
 
 comment "Advanced mbed TLS settings"
 
+config MBEDTLS_AES_ROM_TABLES
+	bool "Use AES ROM tables"
+	depends on VANILLA_MBEDTLS_AES_C || (MBEDTLS_VANILLA_BACKEND && MBEDTLS_AES_C)
+	help
+	  AES lookup tables will be placed in ROM instead of RAM
+	  Placing the AES lookup tables in ROM will perform slower but will reduce RAM usage.
```

Just noticed the mbedTLS config file is very specific regarding RAM savings / ROM usages.

```
 * Tradeoff: Using precomputed ROM tables reduces RAM usage by ~8kb
 * (or ~2kb if \c MBEDTLS_AES_FEWER_TABLES is used) and reduces the
 * initialization time before the first AES operation can be performed.
 * It comes at the cost of additional ~8kb ROM use (resp. ~2kb if \c
 * MBEDTLS_AES_FEWER_TABLES below is used), and potentially degraded
 * performance if ROM access is slower than RAM access.
```
so I think it would be a nice gesture to users to include a line as:
```suggestion
   Placing the AES lookup tables in ROM will perform slower but will reduce RAM usage.
   Using precomputed ROM tables reduces RAM usage by ~8kB with the cost of additional ~8kB ROM use.
```